src/insights_client/__init__.py

```diff
@@ -16,8 +16,8 @@
 from distutils.version import LooseVersion
 
 
-INSIGHTS_DEBUG = os.environ.get("INSIGHTS_DEBUG", "").lower() == "true"
-NO_COLOR = os.environ.get("NO_COLOR") is not None
+LOG_FORMAT = "%(asctime)s %(levelname)8s %(name)s:%(lineno)s %(message)s"
+INSIGHTS_VERBOSE = os.environ.get("INSIGHTS_VERBOSE", "").lower() == "true"
```

Comment:
Since the Core detects both this and `--verbose` flag, should we do the same here?

```py
INSIGHTS_VERBOSE = os.environ.get(...) == "true" or "--verbose" in sys.argv
```

Reply:
The introduced refactoring resolved this concern.